includes/bfcache-opt-in.php

```diff
@@ -492,3 +498,34 @@ function filter_nocache_headers( $headers ): array {
 	__NAMESPACE__ . '\filter_nocache_headers',
 	1000
 );
+
+
+/**
+ * Conditionally adds the Permissions-Policy header to enable BFCache.
+ *
+ * This function checks if the 'bfcache_block_unload_events' option is enabled.
+ * If it is, the Permissions-Policy header with 'unload=()' is added to the HTTP
+ * response. This prevents 'unload' event listeners from firing, allowing the
+ * browser's Back/Forward Cache (BFCache) to function and improve navigation
+ * performance.
+ *
+ * @param array<string, string>|mixed $headers An array of HTTP headers to be sent.
+ * @return array<string, string> The filtered array of headers.
+ * @since n.e.x.t
+ */
+
+
+function disallow_unload_events_permissions_policy_header( $headers ): array {
+    // Get the value of the 'bfcache_block_unload_events' option from the database.
+    $block_unload = get_option(BFCACHE_DISALLOW_UNLOAD_KEY);
+
+    // Check if the value is set to ON
+    if ($block_unload === '1') {
+        // Add the Permissions-Policy header to the array.
+        $headers['Permissions-Policy'] = 'unload=()';
+    }
+
+    return $headers;
+}
+
+add_filter('wp_headers', __NAMESPACE__ . '\disallow_unload_events_permissions_policy_header', 1001);
```

Comment:
Is the super high priority needed?

```suggestion
add_filter('wp_headers', __NAMESPACE__ . '\disallow_unload_events_permissions_policy_header');
```